os/drivers/bluetooth/virtual_ble/ble_vdev.c

```diff
@@ -1,6 +1,6 @@
 /****************************************************************************
  *
- * Copyright 2021 Samsung Electronics All Rights Reserved.
+ * Copyright 2022 Samsung Electronics All Rights Reserved.
```

Comment:
If this file was originally created in 2021, then we must not change the copyright year.

Reply:
Done